src/test.py

```diff
@@ -0,0 +1,603 @@
+import unittest
+import roslibpy
```

Comment:
Would be nice to add a timeout: https://pypi.org/project/timeout-decorator/

Reply:
Added. This does cause some issues with the server tho... Although I think our server should be robust to this so its okay for now.